index.html

```diff
@@ -92,190 +99,113 @@ <h3>LET'S CONNECT</h3>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-          </div>
-          <div class="language">
-            <ul>
-              <li>
-                <p>HTML</p>
-              </li>
-              <li>
-                <p>CSS</p>
-              </li>
-              <li>
-                <p>Javascript</p>
-              </li>
-            </ul>
-          </div>
-          <div class="seeMore">
-            <a href="/"><h3>See project</h3></a>
-          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tempLang">
+            <li class="luga">Html</li>
+            <li class="luga">Javascript</li>
+            <li class="luga">Css</li>
+        </ul>
+        <h3 class="seePro">See project</h3>
         </div>
-      </div>
+      </div> 
 
-      <div id="grid" class="card-one">
-        <div class="card-one">
-          <img src="Icon and img/history Portfolio.png" alt="" />
-          <h3>Multi-Post Stories</h3>
-          <ul>
-            <li>
-              <h4>Canopy</h4>
-            </li>
-            <li>
-              <img src="Icon and img/Counter.png" alt="" />
-            </li>
-            <li>
-              <p>Band End Dev</p>
-            </li>
-            <li>
-              <img src="Icon and img/Counter.png" alt="" />
-            </li>
-            <li>
-              <p>2015</p>
-            </li>
+      <div class="template"> 
+        <div class="templtaeDetails">
+          <h1>Uber Navigation</h1>
+          <ul class="tempUl">
+              <li>Canopy</li>
+              <li>Back-end dev</li>
+              <li>2015</li>
           </ul>
-          <div class="textTonic">
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-          </div>
-          <div class="language">
-            <ul>
-              <li>
-                <p>HTML</p>
-              </li>
-              <li>
-                <p>CSS</p>
-              </li>
-              <li>
-                <p>Javascript</p>
-              </li>
-            </ul>
-          </div>
-          <div class="seeMore">
-            <a href="/"><h3 class="seeMoring">See project</h3></a>
-          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tempLang">
+            <li class="luga">Html</li>
+            <li class="luga">Javascript</li>
+            <li class="luga">Css</li>
+        </ul>
+        <h3 class="seePro">See project</h3>
         </div>
-      </div>
+        <div class="templateImg">
+          <img src="DeskImg/3Portfolio.png" alt="">
+        </div>
+      </div> 
+
     </section>
 
+    
     <section class="about-me">
```

Comment:
- [ ] Please I highly recommend sticking to the required [Figma Desktop Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=XWnQyP1APhJu8dzl-0). 😃

_**REQUIRED FIGMA DESIGN**_

![Screenshot (271)](https://user-images.githubusercontent.com/78104544/236271136-2cb52eb8-75cc-45a9-a24a-1bfb540dd7a6.png)


_**YOUR DESIGN**_

- [ ] Please as numbered as 1, I highly suggest making changes to the `About Myself` text. Please change the `h1` to `h2` - `h6` tags to implement HTML semantics. 😄
- [ ] Please as numbered as 2, I highly suggest adding some width to the `p` tag of the `aboutme` class to fit the required figma design. 😄 
- [ ] Please as numbered as 3, I highly recommend making changes to the text, `LET'US CONNECT`
 to `LET'S CONNECT`. 😄 
- [ ] Please as numbered as 4, I highly suggest making changes to the `resum` class which is a button and needs hover effects per the project requirements. Also makes changes to the text `Get my resume` to `Get my resume`. 😄 
- [ ] Please as numbered as 5. I highly suggest making changes to the `li` in `skills` class which has the text of "Frameworks" and "Skills" respectfully. 😄 
- [ ] Please as numbered as 6, I highly suggest making changes to the `arrow-icons` to fit the required figma design. 😄 


![Screenshot (273)](https://user-images.githubusercontent.com/78104544/236272053-496f7f69-0e5b-4c07-bcda-45e46920770c.jpg)